templates/web-application/webpack.build.js

```diff
@@ -0,0 +1,78 @@
+// @ts-check
+import path from "node:path";
+import HtmlWebpackPlugin from "html-webpack-plugin";
+import TerserPlugin from "terser-webpack-plugin";
+
+import { loadSwcConfig } from "./loadSwcConfig.js";
+
+/** @type {import("webpack").Configuration} */
+export default {
+    mode: "production",
+    target: "web",
+    entry: "./src/index.tsx",
+    output: {
+        path: path.resolve("dist"),
+        // The trailing / is very important, otherwise paths will not be resolved correctly.
+        publicPath: "http://localhost:8080/",
+        clean: true
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(js|ts|tsx)$/i,
```

Comment:
Is there a specific reason to `.js` files here?

Reply:
I'm not sure... I don't see why it should be required or if it can be removed...

In the demo app, it doesn't seem to be required, but maybe in a more complexe one it will be required.

I will go with removing it for now.